migrations/20170627152018-create-author.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Authors', {
+    id: {
+      primaryKey: true,
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.UUIDV4,
+    },
+    firstName: {
```

Comment:
Dat zouden we dan moeten herzien. Als je je column-names gaat snake-casen dan gaan we altijd de db output moeten parsen naar camelCase en creëer je extra ruimte voor errors